MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
Isn't the entire argument of DesktopGL that people shouldn't be writing platform specific code?

They should be using `#if PLATFORM_WINDOWS` and the equivalent for other platforms if they want to write platform-specific code.

> This returns the current OS, and not the platform, WindowsDX, DesktopGL, UWP and Metro can all return Windows as OS.

This feels pretty arbitrary, since the native APIs available on a phone under UWP are going to be different to the native APIs available under Win32.